redis/endpoint.bal

```diff
@@ -70,7 +76,7 @@ public client class Client {
     # + return - Size of the string stored in the destination key, that is equal to the size of the longest input
```

Comment:
The size of the string stored in the destination key, which is equal to the size of the longest input